modules/test/playwright/tests/object-web/objectFields.spec.ts

```diff
@@ -162,3 +162,90 @@ test.describe('Manage object fields through Model Builder', () => {
 		).toBeVisible();
 	});
 });
+
+test.describe('Manage objectFields through Objects Admin UI', () => {
+	test('cannot delete an objectField that belongs to a unique composite key validation through Objects Admin UI', async ({
+		apiHelpers,
+		objectFieldsFDSPage,
+		page,
+		viewObjectDefinitionsPage,
+	}) => {
+		const integerFieldName = 'integerField' + getRandomInt();
+
+		await apiHelpers.objectAdmin.postObjectFieldByExternalReferenceCode(
+			objectDefinition.externalReferenceCode,
+			{
+				DBType: 'Integer',
+				businessType: 'Integer',
+				externalReferenceCode: integerFieldName,
+				indexed: true,
+				indexedAsKeyword: false,
+				indexedLanguageId: '',
+				label: {en_US: integerFieldName},
+				listTypeDefinitionId: 0,
+				localized: false,
+				name: integerFieldName,
+				readOnly: 'false',
+				required: false,
+				state: false,
+				system: false,
+			}
+		);
+
+		const objectValidationName =
+			'Unique Composite Key Object Validation' + getRandomInt();
+
+		await apiHelpers.objectAdmin.postObjectValidation(
+			objectDefinition.externalReferenceCode,
+			{
+				active: true,
+				engine: 'compositeKey',
+				engineLabel: 'Composite Key',
+				errorLabel: {
+					en_US: 'Unique composite key object validation error',
+				},
+				name: {
+					en_US: objectValidationName,
+				},
+				objectValidationRuleSettings: [
+					{
+						name: 'compositeKeyObjectFieldExternalReferenceCode',
+						value: 'textField',
+					},
+					{
+						name: 'compositeKeyObjectFieldExternalReferenceCode',
+						value: integerFieldName,
+					},
+				],
+				outputType: 'fullValidation',
+				script: '',
+				system: false,
+			}
+		);
+
+		await viewObjectDefinitionsPage.goto();
+
+		await expect(
+			page.locator(`a:has-text("${objectDefinition.label['en_US']}")`)
+		).toBeVisible();
+
+		await viewObjectDefinitionsPage.editObjectDefinitionFDSLink(
+			objectDefinition.label['en_US']
+		);
+
+		await objectFieldsFDSPage.goto();
+
+		const newObjectFieldLink = page.getByText(integerFieldName);
+
+		await expect(newObjectFieldLink).toBeVisible();
+
+		await objectFieldsFDSPage.deleteObjectField(-1);
```

Comment:
Hey @danvb112, I can't undestand why you use -1 here 😆, could you explain to me?